doc/METADATA.ini

```diff
@@ -1211,6 +1211,13 @@ description= check if value is a valid rgbcolor in hexformat
 # Old stuff inherited from filesys plugin
 #
 
+[check/memoryvalue]
```

Comment:
"memoryvalue" is not really a well known term. Ideally you would rename it to `check/unit` and also allow other units, and not only "B".

Reply:
In that case this PR should be merged with #2614

Reply:
In #2614 the author stopped with CM.

So this PR could use check/unit.

Reply:
should I rename it to, check/unit?